### PR TITLE
Fix occasional bus port conflict issue

### PR DESCRIPTION
**Pull request by chunfuwen**
Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
@balamuruhans 
@will-Do 
Could you please take a look?

**Comment by chunfuwen** [comment]
@balamuruhans 

Could you please take a look?

File: libvirt/tests/cfg/virtual_disks/virtual_disks_multidisks.cfg [hunk 1]
```diff
@@ -562,7 +562,7 @@
                                         - default:
                                             add_usb_device = "no"
                                         - normal_test:
-                                            virt_disk_addr_options = "type=usb,bus=1,port=2"
+                                            virt_disk_addr_options = "type=usb"
```
Comment (will-Do) [inline libvirt/tests/cfg/virtual_disks/virtual_disks_multidisks.cfg]: So this will not affect the usage of 'virt_disk_addr_options', right?

Comment (chunfuwen) [inline libvirt/tests/cfg/virtual_disks/virtual_disks_multidisks.cfg]: Yes, it should not. Libvirt will auto-fit bus and port attributes if absent. Previous hardcoded those attributes can see some resource conflict in testing result.

**Comment by chunfuwen** [comment]
@will-Do 
Please help take a look.

**Pull request merged**